### DISCUSSION TITLE
Étude de cas : "Histoire de l'art"

### DISCUSSION THREAD

**Discussion by MCtzr** [Hypertopic/HyperGlosae#133] [Show and tell]
Here is the video presenting all the content we added to Hyperglosae during this semester.

https://github.com/Hypertopic/HyperGlosae/assets/91234392/70bad373-7ab2-4a96-be4b-f501b890826c

**Comment by benel** [comment]
Merci à toute l'équipe pour votre contribution aux expérimentations de M. Fallot et au logiciel lui-même.

Je laisse la parole à @adrienfallot pour des commentaires ou des questions.

**Comment by philacour** [comment]
Pertinent, bravo

**Comment by adrienfallot** [comment]
Bonjour à tous,

Tout d’abord, merci beaucoup pour votre travail durant ce semestre ! Vous avez su cerner les besoins à la suite de notre première entrevue et implémenter les fonctionnalités correspondantes.

Grâce à votre travail, l’application a été utilisée lors d’une [visite de l’église Saint-Nicolas à Troyes](https://d1h122cknc6elf.cloudfront.net/collection/1451decd-68ad-406f-9293-f86d59fc725c/document/e5a92c28-0699-11ee-a942-367ddac73ffe). Malgré quelques bugs résiduels (#137), les visiteurs ont apprécié l’utilisation de parcours pour découvrir l’édifice et pouvoir changer de collection et découvrir en détail certains vitraux. Comme j’espérais, la comparaison de vitraux similaire au sein d’une même collection à bien permet d’en faire ressortir les éléments clés.

Avez-vous eu l’occasion de tester l’application sur place ? Car bien que les visiteurs apprécient l’utilisation des parcours, il y a toujours quelques problèmes d’ergonomie lors au passage sur mobile. Notamment pour changer de collection, seul le logo est cliquable, et non toute la boite grise. 
De plus, pour le futur, de nombreux visiteurs regrettent l’absence de menu répertoriant l’ensemble des parcours disponibles pour cet édifice. Pensez-vous cela possible ? 

Merci encore pour votre travail !
